### PR TITLE
Fix time shift bug

### LINKED ISSUE #150: My comment is shown as 2 hours ago despite it was posted just now
See the screenshot.

The APK is again the one taken from issue #138.

### PR DESCRIPTION
**Pull request by krawieck**
fixes #150 

the bug was that time from server is in UTC format, and our time is in local format, so it needed to be shifted back to UTC.

oddly enough `.toUtc()` method didn't work

also added a `DateTime` extension for easier use

### CODE REVIEW

File: lib/util/extensions/datetime.dart [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import 'package:timeago/timeago.dart' as timeago;
+
+extension FancyTime on DateTime {
+  /// returns [this] time as a relative, human-readable string. In short format
```
Comment (shilangyu) [inline lib/util/extensions/datetime.dart:4]: `[this]` doesn't link to anything

File: lib/util/extensions/datetime.dart [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  String get fancyShort => timeago.format(this,
+      locale: 'en_short',
+      clock: DateTime.now().subtract(DateTime.now().timeZoneOffset));
+
+  /// returns [this] time as a relative, human-readable string
+  String get fancy => timeago.format(this,
```
Comment (shilangyu) [inline lib/util/extensions/datetime.dart:10]: I really don't understand why the word 'fancy' was chosen

**Review by shilangyu — APPROVED** [review]

File: lib/util/extensions/datetime.dart (hunk 1 above)
Comment (shilangyu) [reply lib/util/extensions/datetime.dart:4]: Nevermind, I'm stupid

**Comment by dessalines** [comment]
I don't know about darts fromUtc libraries, but this should work : https://stackoverflow.com/a/60188960

File: pubspec.yaml [hunk 3]
```diff
@@ -44,7 +44,8 @@ dependencies:
   # utils
   timeago: ^2.0.27
   fuzzy: <1.0.0
-  lemmy_api_client: ^0.10.2
+  lemmy_api_client: ^0.11.0
+
```
Comment (shilangyu) [inline pubspec.yaml:48]: :thinking:

**Review by shilangyu — CHANGES_REQUESTED** [review]

**Pull request merged**